.github/workflows/ci_compliance.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Compliance
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  duvet:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: goatgoose/duvet-action@v1.0.0
+        with:
+          report-script: compliance/generate_report.sh
```

Comment:
When we talked about this, you mentioned not committing all the spec files and just downloading them as part of the action. Why commit them?

Reply:
The nice thing about committing them is the check is completely offline. This, for example, removes the requirement of the IETF site to be available.

Reply:
Originally I wasn't really sure which to do. I liked the idea of not needing to commit all of the spec files, but also liked that extra steps wouldn't be needed to generate the report locally, since I was planning on having the RFCs be an input to the action. However, thinking about it now, I probably could just include that part in the script passed to the action, so it would generate them locally also. So maybe this is an option? The offline aspect of committing them does makes sense though.